README.md

```diff
@@ -38,6 +40,26 @@ This assumes your mocks are placed in the mocks directory which is located at th
 In the mocks directory you have exports for either complete scenarios or separate mock requests.
 
 
+Guides
+-------------
+
+* [Using mimic with React Native](https://github.com/500tech/mimic/blob/master/docs/react-native.md)
+* [Using mimic with NativeScript](https://github.com/500tech/mimic/blob/master/docs/nativescript.md)
+
+
+Other Usages
+------------
+The main and the most obvious use case for mimic is when the API for a UI feature is not ready or incomplete,
+but it can be helpful in more situations. For example:
+
+* You have a demo of the product, but the API server is unstable.
+* You want to work on a feature outside of company's VPN
+* You need access to API, but don't have an internet connection or it is unstable.
```

Comment:
I think it's also worth mentioning the product demo with different marketing scenarios to showcase the product. this would be a very useful feature imo in places where you sell a data based product (like analytics) and want to showcase it to clients with attractive data that doesn't belong to any client.